src/dapp/libexec/dapp/dapp-verify-contract

```diff
@@ -113,8 +113,8 @@ while [ $count -lt 5 ]; do
   response=$(curl -fsS "$ETHERSCAN_API_URL" -d "$query" \
   --data-urlencode "compilerversion=$version" \
   --data-urlencode "sourceCode@"<(echo "$source") \
-  --data-urlencode "constructorArguements=$constructorArguments" -X POST)
-  # NOTE: the Arguements typo is in etherscan's API
+  --data-urlencode "constructorArguments=$constructorArguments" -X POST)
+  # NOTE: the Arguments typo is in etherscan's API
```

Comment:
This typo is part of the etherscan api

Reply:
Crazy lmao

Reply:
Ha wow, just checked hardhat-etherscan to confirm: https://github.com/nomiclabs/hardhat/blob/767f68dee84d6a47cd40153a8ab8552f41a95d5e/packages/hardhat-etherscan/src/etherscan/EtherscanVerifyContractRequest.ts#L14-L16

Reply:
and in the official api docs here: https://docs.etherscan.io/api-endpoints/contracts#verify-source-code

Reply:
What the hell.